frontend/server/src/Controllers/Problem.php

```diff
@@ -6162,18 +6178,19 @@ public static function getCollectionsDetailsByAuthorForSmarty(\OmegaUp\Request $
             /*$onlyQualitySeal=*/true,
             /*$url=*/'/problem/collection/author/',
             /*$level=*/null,
-            $difficulty
+            $difficulty,
+            $authors
         );
 
-        $authors = \OmegaUp\Controllers\User::getAuthorsRankWithQualityProblems(
+        $authorsRanking = \OmegaUp\Controllers\User::getAuthorsRankWithQualityProblems(
             /*$offset*/            1,
             /*$rowCount*/15
         );
 
         return [
             'smartyProperties' => [
                 'payload' => [
-                    'authors' => $authors,
+                    'authors_ranking' => $authorsRanking,
```

Comment:
nit:

```suggestion
                    'authorsRanking' => $authorsRanking,
```

poco a poco estamos evitando el uso de guiones bajos excepto como parámetros / resultados del API (las funciones que empiezan con `api` y no terminan con `ForSmarty`) y SQL.